group_fixup.py

```diff
@@ -276,6 +276,22 @@ def provision_group(gid):
     return call_api3(POST, path, data)
 
 
+def provision_group_members(gid):
+    prov_id = options.prov_id
```

Comment:
```suggestion
def provision_group_members(gid, prov_id):
```

It doesn't look like `options` is global so I think this should be a function arg

Reply:
That bit is a direct copy of `provision_group` (`options` is defined on line 74).

I kinda favor stylistic consistency over the global reference, even though I know of at least one time that this global reference style caused the originally author himself some confusion during testing.

Reply:
😿 I suppose I can live with that